README.md

```diff
@@ -42,83 +24,58 @@ The EzyFox ecosystem supports a wide range of essential components for enterpris
 
 With the EzyFox ecosystem, we can avoid the headache of choosing which technologies to use to manage and scale up an application, allowing us to focus solely on implementing business logic.
 
-# Getting Started
-
-[https://youngmonkeys.org/get-started/](https://youngmonkeys.org/get-started/)
-
-# Documentation
-
-[https://youngmonkeys.org/ezyfox-sever/](https://youngmonkeys.org/project/ezyfox-sever/)
+## ✨ Features
 
-# Latest Version
-
-You can [download it here](https://resources.tvd12.com/)
+* **Core Container & Dependency Injection**: Includes bean manipulation, auto-binding, auto-implementation, and more.
+* **Multiple Communication Protocols**: Supports TCP, UDP, WebSocket, and HTTP.
+* **Traffic Encryption**: Allows for traffic between clients and servers to be encrypted using SSL.
+* **Multiple Client SDKs**: Includes Android, iOS, Unity, React, C++, Flutter, and more.
 
-# Code Example
 
-**1. Create an app entry**
+## 🚀 Getting Started
 
-```java
-public static class HelloAppEntry extends EzySimpleAppEntry {
+[Let's see how we can getting started just in 5 minutes!](https://youngmonkeys.org/get-started/)
 
-    @Override
-    protected String[] getScanablePackages() {
-        return new String[] {
-                "com.tvd12.ezyfoxserver.embedded.test" // replace by your package
-        };
-    }
-}
-```
+## 📑 Documentation
 
-**2. Create a plugin entry**
+- [Official website document](https://youngmonkeys.org/projects/ezyfox-server)
+- [Issue](https://github.com/youngmonkeys/ezyfox-server/issues)
 
-```java
-public static class HelloPluginEntry extends EzySimplePluginEntry {
+## 📐 Architect Overview
+<div align="center">
+    <img src="./images/ezyfox-server-architecture.png" width="512" />
+</div>
 
-    @Override
-    protected String[] getScanablePackages() {
-        return new String[] {
-                "com.tvd12.ezyfoxserver.embedded.test" // replace by your package
-        };
-    }
+-----------
 
-}
-```
+- A EzyFox Server will contain multi zones
+- A zone will contain zone's user manager mutl apps and multi plugins
+- An App will contain app's user manager
+- A Plugin just handle events and client's requests
```

Comment:
- An EzyFox Server will contain multiple zones.
- Each Zone will contain a zone's user manager, multiple apps, and multiple plugins.
- An App will contain an app's user manager.
- A Plugin will only handle events and client's requests.

Reply:
Okay, I will update soon.
@tvd12 please update on your website at here: https://youngmonkeys.org/ezyfox-server/guides/ezyfox-server-architecture

Reply:
updated, thank you

Reply:
```suggestion
- An EzyFox server contains multiple zones
- A zone contains its user manager, multiple apps and multiple plugins
- An app contains its user manager
- A plugin only handles events and clients' requests
```

Reply:
@trungdlp-wolffun could you update by Vu suggestion? I see it's more suitable

Reply:
@tvd12 I updated.